### ISSUE TITLE
"no device names" and "big numbers" do the same thing

### ISSUE THREAD

**Issue by Soukyuu** [kotelnik/plasma-applet-network-monitor#8]
I think this might be a bug. I expected "no device names" look like "full view" but without, well, the device names. However, selecting it makes the up/down arrows disappear and the number grow bigger. Setting  it to "big numbers" shifts the numbers a bit, but the size stays the same.

**Comment by kotelnik** [comment]
I cannot reproduce this behaviour. When I change setting from "no device names" to "big numbers" I can see that it not only shifts the numbers but also made them bigger.

But maybe you've found a specific use-case where this size change is not visible?

**Comment by Soukyuu** [comment]
Uh, sorry, I kind of forgot about this issue. This is how the widget looks for me in different modes:
![nm_full](https://cloud.githubusercontent.com/assets/2095054/12374830/5916d4a0-bca9-11e5-97a2-3c7a9d3897e2.jpg)
![nm_nodev](https://cloud.githubusercontent.com/assets/2095054/12374831/62173a54-bca9-11e5-81ec-8e9a4795581c.jpg)
![nm_bignum](https://cloud.githubusercontent.com/assets/2095054/12374832/67df6312-bca9-11e5-8612-9bf7df0b9fda.jpg)
![nm_oneline](https://cloud.githubusercontent.com/assets/2095054/12374833/6afa9904-bca9-11e5-9412-040e594958da.jpg)

Note how everything except full view has weird spacing, most notably the "one line" mode. "big numbers" and "no device names" look nearly identical. I suspect it has something to do with me using a vertical panel though.

**Comment by kotelnik** [comment]
I can see you have a little bit different spacing then me but still "big numbers" was created from "no device names" to preserve more space for showing bigger numbers with container being wider (not squared). From the screenshots I can see that "big numbers" actually preserves a little vertical space in vertical panel.

The weird spacing in "full view" is caused by the need to keep the font smaller so higher numbers are not overflowing the arrows.

One line is mostly made for horizontal panels.

But maybe I'm not answering the right points you want me to... Please feel free to suggest some changes or ideas :)

**Comment by Soukyuu** [comment]
Basically, I have expected picture 2 to look like picture 1 (small text, arrows), but without the "enp5s0" text. There is almost no difference between picture 2 and 3 as it is now, at least for me.

**Comment by kotelnik** [comment]
You are right they look very similar. They was created on some users' demands to show bigger numbers. I think all 4 options are used by users so I'm probably going to leave them there of the time being.
